cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -282,6 +300,112 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 			// imageName := t.ImageName
 			// projectID := gcp.GetProjectID()
 
+		case *target.AzureImageTargetOptions:
+			ctx := context.Background()
+
+			var authorizer autorest.Authorizer
+			if impl.AzureCreds == nil {
+				authorizer, err = auth.NewAuthorizerFromEnvironment()
```

Comment:
The idea is that this would use an Azure Managed Identity if no environment variables are set, right?

Reply:
I've decided to remove this feature for now. With `auth.NewAuthorizerFromEnvironment()`, we cannot specify the tenant ID per job. This should surely be possible but needs more research and it's actually not needed for the MVP.